test/functional/apps/visualize/_tsvb_chart.js

```diff
@@ -76,7 +76,40 @@ export default function ({ getService, getPageObjects }) {
       });
     });
 
-    describe('Visual Builder markdown', () => {
+    // add a gauge test
+    describe('gauge', () => {
+      it('should click on gauge viz', async function () {
```

Comment:
I might have confused you earlier when we discussed `before` method vs `it` tests.  This shouldn't be a test because it's not verifying anything.  It could be a `before` within the `gauge` describe block.  Or it could just be part of the next test.

Same with `topN` below.